src/Adapters/BlobAdapter.ts

```diff
@@ -175,7 +175,9 @@ export default class BlobAdapter implements IBlobAdapter {
                         headers: this.generateBlobHeaders(headers)
                     });
                     if (scenesBlob.data) {
```

Comment:
This should check for **scenesBlob.data && scenesBlob.data.data**

Reply:
done!